packages/datetime2/package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "@blueprintjs/datetime2",
-    "version": "0.2.0",
+    "version": "0.3.0",
```

we only bump versions during release. this needs to be reverted.

```suggestion
    "version": "0.2.0",
```